packages/apps/devcon/changelog.md

```diff
@@ -1,3 +1,6 @@
+1.0.87
+- show time in 24h format for all locales
```

Comment:
todo: this change is not good

somehow merging or rebasing went wrong.. these changes should not be part of your PR